HOWTO

```diff
@@ -2551,6 +2551,12 @@ I/O latency
 	defaults to 100.0, meaning that all I/Os must be equal or below to the value
 	set by :option:`latency_target`.
 
+.. option:: latency_run=bool
+
+	If not set, fio will find the highest queue depth that meets
+	:option:`latency_target` and exit. If set, fio will continue running and
+	try to meet :option:`latency_target` by addjusting queue depth.
```

Comment:
Typo - (you have two ds in `addjusting` when you only need one). Can you also put what does this option defaults to?

Reply:
Will fix. Thanks!